_posts/en/newsletters/2019-09-04-newsletter.md

```diff
@@ -0,0 +1,131 @@
+  step (*receiver* step) by periodically checking the server to see if
+  anyone like Alice has sent him a proposed PSBT.  If so, Bob can
+  evaluate the PSBT to ensure it's correct, add his signature for his
+  UTXO to finalize it, and broadcast the transaction to complete the
+  coinjoin.
+
+  The key advantage of this proposal is that no interaction is required
+  between Alice and Bob.  They each perform their steps independently
+  and aren't limited by having each other as potential partners.  Alice
+  can create as many proposals as she wants at no cost (except server
+  storage space) and Bob can receive multiple proposals from different
+  people via a variety of servers, selecting whichever proposal he
+  prefers (or none at all).  Either party can also spend their UTXO
+  normally at any time, automatically invalidating any pending proposals
+  without any harm done.  The PSBTs can be exchanged using any medium,
+  such as a via a simple FTP server, making it easy for anyone to host a
+  SNICKER exchange server.
+
+  The main downside of the proposal is that it requires the proposer
+  (Alice) know the public key of the receiver (Bob).  Almost all
+  transactions today pay an address that doesn't include a public key,
+  although that may to change if the proposed [taproot][bip-taproot]
+  soft fork is activated and becomes widely adopted.  In the meantime,
+  the SNICKER proposal suggests using reused addresses where public keys
+  have been revealed via the block chain, or by using a public key
+  from the input of a transaction that creates a UTXO.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [C-Lightning #3002][] makes several small changes to comply with
+  recent updates to the [LN specification][BOLTs], including the
+  [BOLT4][] update to remove the `final_expiry_too_soon` error message
+  as described in [last week's newsletter][bolts608].
+
+- [Eclair #899][] implements extended queries as proposed in [BOLTs
+  #557][], allowing an LN node to only request gossip updates that
+  arrived after a certain time or which have a different checksum than
+  those the node has already seen.
+
+- [Eclair #954][] adds a sync whitelist.  If empty, the node will sync
+  its gossip store with any peer. If not empty, the node will only sync
+  with the specified peers.
+
+{% include linkers/issues.md issues="3002,899,557,954" %}
+
+[cve ln]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-August/002130.html
+[bolts608]: /en/newsletters/2019/08/28/#bolts-608
+[bolts]: https://github.com/lightningnetwork/lightning-rfc/
+[snicker]: https://gist.github.com/AdamISZ/2c13fb5819bd469ca318156e2cf25d79#storage-of-keys
```

Comment:
Is the #storage-of-keys part of this URL desired? IIUC the link is used in the intro sentence "Adam Gibson posted to the Bitcoin-Dev mailing list with a [proposal][snicker] for *Simple Non-Interactive Coinjoin with Keys for Encryption Reused*"...